### PR TITLE
Create the process and initial proposal.

### PR DESCRIPTION
**Pull request by MarkLodato**
Part of https://github.com/slsa-framework/slsa/issues/296.

A separate PR to the main REPO updates the CONTRIBUTING file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,63 @@
...
+
+## How to create a proposal document
+
+See [CONTRIBUTING] for a description of the process. Proposal documents are not
+needed for all changes.
+
+1.  Find or create a relevant [GitHub Issue] in the main SLSA repo to discuss
+    this idea.
+2.  Fork this repository and create a PR containing the new document:
+    *   Create `####/index.md` using [0001/README.md](0001/README.md) as a
```
Comment (joshuagl) [inline README.md]:
```suggestion
    *   Create `####/README.md` using [0001/README.md](0001/README.md) as a
```
I assume we want the proposal to README.md, like proposal 0001, not index.md?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,63 @@
...
+3.  As needed, iterate on the proposal with more PRs.
+4.  Once agreement is reached on the Issue, send a PR to update the [status].
+
+## Meaning of status codes
+
+*   **DRAFT:** Still being refined and/or undergoing review. All proposals start
+    in this state.
+*   **ACCEPTED:** Accepted as the plan of record but not yet implemented.
+*   **IMPLEMENTED:** Implemented and final. The doc should be point to the
+    implementation PRs.
```
Comment (joshuagl) [inline README.md]: missing some words, perhaps:
```suggestion
*   **IMPLEMENTED:** Implemented and final. The doc should be updated to point to
    the implementation PRs in the Implementation metadata field.
```

**Review by joshuagl — APPROVED** [review]
Thanks Mark. Couple of minor nits, but this looks like a great starting point.

File: README.md (hunk 1 above)
Comment (MarkLodato) [reply README.md]: Oops, typo, thanks for catching. (Forgot to revert from an earlier GH Pages-based version.)

File: README.md (hunk 2 above)
Comment (MarkLodato) [reply README.md]: Oops, done.

**Comment by trishankatdatadog** [comment]
So sorry: going to be very difficult for me to review right now as I plan to travel, so I will defer to @joshuagl. Thanks very much!

**Comment by MarkLodato** [comment]
> So sorry: going to be very difficult for me to review right now as I plan to travel, so I will defer to @joshuagl. Thanks very much!

No problem!

**Comment by trishankatdatadog** [comment]
OK, now that things have settled down a bit, I should be able to review this weekend or so... 🤞🏽

**Comment by MarkLodato** [comment]
If there are no objections I'll merge today or tomorrow.

**Pull request merged**